exporter/splunkhecexporter/factory.go

```diff
@@ -57,6 +58,20 @@ func NewFactory() exporter.Factory {
 		exporter.WithLogs(createLogsExporter, metadata.LogsStability))
 }
 
+func getExporterOptions(cfg *Config, startFunc component.StartFunc, shutdownFunc component.ShutdownFunc) []exporterhelper.Option {
```

Comment:
This extraction is unrelated now. Better just to add `exporterhelper.WithBatcher(cfg.BatcherConfig),` lines separately to keep the changelog clean

Reply:
Sure. put back optional to inline